api/api.go

```diff
@@ -266,13 +278,7 @@ func configureExtensionsApi(config *core.Config, router *mux.Router, apiRoot str
 		)
 	}
 
-	api.HandleFunc(path.Join(apiRoot, "{uuid:(?:[a-z]|[0-9]|-)+\\/?}"), handlerWithCors(api.extensionRootHandler))
-
-	api.PathPrefix("/dev-console").Handler(http.FileServer(http.FS(devConsole)))
-
-	api.PathPrefix("/dev-assets/").Handler(http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
-		http.Redirect(rw, r, path.Join("/dev-console", r.URL.Path), http.StatusTemporaryRedirect)
-	}))
+	api.HandleFunc(path.Join(apiRoot, "{uuid:(?:[^a-z][a-z]|[0-9]|-)+\\/?}"), handlerWithCors(api.extensionRootHandler))
```

Comment:
Is this change to force UUID detection?
This is not working with the current CLI as we use the extension name and a random string as `dev uuid`, like:
`ext-01-50ovte7`, it's more user friendly than `00000000-0000-0000-0000-000000000002`